include/llvm/Transforms/Scalar.h

```diff
@@ -460,7 +460,7 @@ FunctionPass *createPlaceSafepointsPass();
 // RewriteStatepointsForGC - Rewrite any gc.statepoints which do not yet have
 // explicit relocations to include explicit relocations.
 //
-ModulePass *createRewriteStatepointsForGCPass(bool TrackBasePointers = true);
+ModulePass *createRewriteStatepointsForGCPass();
```

Comment:
This TrackBasePointers change doesn't affect LLVM mainline. Its a change to MS branch.